content/_data/upgrades/2-221-1.adoc

```diff
@@ -0,0 +1,76 @@
+==== Header and Breadcrumb Layout Improvements
+
+Instances with plugins that depend on specific details of the Jenkins layout (like the https://plugins.jenkins.io/simple-theme-plugin/[Simple Theme plugin]) may have user interface issues with this release.
+In the case of the Simple Them Plugin, users or theme maintainers may need to update CSS and JavaScript to match the new layout structure.
+
+==== Removed Agent Protocols
+
+Remoting changelogs for https://github.com/jenkinsci/remoting/releases/tag/remoting-4.2[4.2],
```

Comment:
If you want to list all versions, IMHO it should be in the changelog, not in the upgrade guide. Here we need to provide links to items related to the upgrade entry, not to make admins to go through all changelogs which are mostly unrelated